pygmt/src/grdimage.py

```diff
@@ -148,10 +148,14 @@ def grdimage(self, grid, **kwargs):
     no_clip : bool
         Do **not** clip the image at the frame boundaries (only relevant
         for non-rectangular maps) [Default is ``False``].
-    nan_transparent : bool
+    nan_transparent : bool or str
+        [**+z**\ *value*][*color*]
         Make grid nodes with z = NaN transparent, using the color-masking
```

Comment:
> using the color-masking feature in PostScript Level 3 (the PS device must support PS Level 3).

PostScript Level 3 was released in 1997. Is it safe to assume that all printers support PS Level 3 now? Or do we care about PS Level 3 if we always convert PS to PDF then print?

Reply:
Ping @PaulWessel just in case he missed this comment.

Reply:
I think it's fine to merge this keeping the warning